### PR TITLE
feature81 - On hover components button background

### PR DESCRIPTION
**Pull request by AdrianaAC**

### CODE REVIEW

**Comment by AdrianaAC** [comment]
[On hover components button background](https://trello.com/c/8RBCkUGC/81-on-hover-components-button-background)

**Comment by JWebCoder** [comment]
please update the design to be closer to the one from figma

**Review by JWebCoder — CHANGES_REQUESTED** [review]
please update the design to be closer to the one from figma

**Review by miriampereira — APPROVED** [review]

**Review by Felix-by-merkle — CHANGES_REQUESTED** [review]

File: .hintrc
[the remark this answers is not in the document]
Comment (JWebCoder) [reply .hintrc:1]: what is this for @AdrianaAC ?

**Review by JWebCoder — CHANGES_REQUESTED** [review]
review the ticket on trello, we need some fixes to the button colors.

**Review by JWebCoder — APPROVED** [review]

**Pull request merged**